tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -263,7 +263,7 @@ def get_numeric_feature_names(example):
     example: An example.
 
   Returns:
-    A list of string feature names.
+    A list of numeric feature names.
```

Comment:
the feature names that are returned are for the features in the examples that are numeric, so we want to keep the word numeric here. Perhaps "A list of strings of the names of the features that are numeric"?

Reply:
Okay, I initially added numeric and removed string because it was ambiguous to me, but I suppose it is still ambiguous since it's not differentiating between the type of the list (str) and what the list is representing (numeric feature names). I updated the text to add both "string" and "numeric".